src/main/java/uk/gov/hmcts/cmc/claimstore/events/offer/SignSettlementAgreementEvent.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.cmc.claimstore.events.offer;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+@EqualsAndHashCode
```

Comment:
could you swap these around? so that the smaller is on top?

Reply:
I prefer to arrange in alphabetical order